curated_transformers/tokenization/bbpe_encoder.py

```diff
@@ -81,3 +81,16 @@ def byte_bpe_encoder_forward(model: Model, X: List[Doc], is_train: bool):
         )
 
     return pieces, lambda dY: []
+
+
+@registry.model_loaders("curated-transformers.ByteBPELoader.v1")
```

Comment:
Oh, I realize now that these can also be moved to `setup.cfg`, on it. Changing to Draft until I have finished this.

Reply:
Done. Also changed the tests to get the factories through the registry.